internal/providers/pluginfw/products/permission/resource_permission.go

```diff
@@ -0,0 +1,186 @@
+package permission
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/iam"
+	"github.com/databricks/terraform-provider-databricks/common"
+	pluginfwcommon "github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/common"
+	pluginfwcontext "github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/context"
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/tfschema"
+	"github.com/databricks/terraform-provider-databricks/permissions/entity"
+
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+const resourceName = "permission"
+
+var (
+        _ resource.Resource = &permissionResource{}
+        _ resource.ResourceWithConfigure = &permissionResource{}
+)
+
+func NewPermissionResource() resource.Resource {
+    return &permissionResource{}
+}
+
+type permissionResource struct {
+        client *common.DatabricksClient
+        context context.Context
+}
+
+func (r *permissionResource) Configure(ctx context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+        if req.ProviderData == nil {
+                return
+        }
+
+        var provider 
+        //TODO CHECK IF ACCOUNTID ATTRIBUTE IS SET IN PROVIDER CONFIG STRUCT,
```

Comment:
permission is a workspace-level resource, so we should not worry about account-level